Day12/github-api-project-ts/src/routes/index.tsx

```diff
@@ -42,9 +42,11 @@ export const myAppRouter = createBrowserRouter([
   {
     path: "/login",
     element: <Login />,
```

Comment:
error element you can add when you are referencing some backend data for existing user else i don't think this case will arise.

Reply:
Actually, I got an error while loading the Login Page and the same react-router error that I should have an ErrorComponent to handle, so I've added it. Yes, it was an error from Frontend Only.